x/bank/keeper/send.go

```diff
@@ -24,8 +24,11 @@ type SendKeeper interface {
 	SetBalance(ctx sdk.Context, addr sdk.AccAddress, balance sdk.Coin) error
 	SetBalances(ctx sdk.Context, addr sdk.AccAddress, balances sdk.Coins) error
 
-	GetSendEnabled(ctx sdk.Context) bool
-	SetSendEnabled(ctx sdk.Context, enabled bool)
+	GetParams(ctx sdk.Context) types.Params
+	SetParams(ctx sdk.Context, params types.Params)
+
+	GetSendEnabled(ctx sdk.Context, denom string) bool
```

Comment:
```suggestion
	SendEnabledDenom(ctx sdk.Context, denom string) bool
```

Reply:
Related to above, method name order was changed however current implementation unifies on Coin as the parameter type in order to match all the other bank send keeper methods and improve developer ergonomics.